docs/spec/staking/end_block.md

```diff
@@ -1,68 +1,40 @@
 # End-Block 
 
-## Unbonding Validator Queue
+## Validator Set Changes
```

Comment:
Maybe an intro here to explain whats going on in this file? We jump straight into `Validator Set Changes` with no other explainations.

Reply:
k